string sign_type
float64 type_confidence
int32 xmin
int32 ymin
int32 xmax
int32 ymax
